qvm/template-whonix-gw.sls

```diff
@@ -11,20 +11,22 @@
 #   qubesctl state.sls qvm.template-whonix-gw dom0
 ##
 
-template-whonix-gw:
+{%- import "qvm/whonix.jinja" as whonix -%}
```

Comment:
Having `-` on both ends cause the first state being concatenated with the comment above (`##template-whonix-gw-...`).